compiler/codegen/NodeEvaluation.cpp

```diff
@@ -170,11 +170,7 @@ OMR::CodeGenerator::evaluate(TR::Node * node)
       // 2) destructive evaluation by either class of function must be possible.
       //
       // This code implements changes to eliminate the second condition for this bug by performing
-      // the following check and fixup:
-      //
-      // If in a function which partially evaluates its subtree, note all non-restricted nodes that
-      // have a reference count > 1. If any of those node's reference counts reach 1, then artificially
-      // inflate those reference counts by 1 for the lifetime of the parent evaluation.
+      // the following check and fixup.
```

Comment:
I believe the bulk of the original paragraph still applies.  It think all that's required to remove PLX-isms is to remove the words "non-restricted" from the original comment.